README.md

```diff
@@ -1 +1,159 @@
 # testdrive
+
+`testdrive` is a library for:
+
+ * Building a URI for a test case from a base URI and the test case path
```

Comment:
It's not so clear from this how URI can be used. I guess it's for the `testdrive.run`? also, how can we build a URI for a test/set of tests?